examples/create-threshold-account.js

```diff
@@ -0,0 +1,49 @@
+const {
+  Client,
+  Ed25519PrivateKey,
+  AccountCreateTransaction,
+  ThresholdKey
+} = require("@hashgraph/sdk");
+
+async function main() {
+  
+  const operatorAccount = process.env.OPERATOR_ID;
+  const operatorPrivateKey = process.env.OPERATOR_KEY;
+
+  if (operatorPrivateKey == null || operatorAccount == null) {
+    throw new Error(
+      "environment variables OPERATOR_KEY and OPERATOR_ID must be present"
+    );
+  }
+
+  const client = Client.forTestnet()
+  client.setOperator(operatorAccount, operatorPrivateKey);
+
+  // Generate our key lists
+  const privateKeyList = [];
+  const publicKeyList = [];
+  for (var i = 0; i < 4; i++) {
+    var privateKey = await Ed25519PrivateKey.generate();
+    var publicKey = privateKey.publicKey;
+    privateKeyList.push(privateKey);
+    publicKeyList.push(publicKey);
+    console.log(i + ": pub key:" + publicKey);
+    console.log(i + ": priv key:" + privateKey);
+  }
+
+  // Create our threshold key
+  const thresholdKey = new ThresholdKey(3); // Define min # of sigs
+  for (var i = 0; i < publicKeyList.length; i++) {
+    thresholdKey.add(publicKeyList[i]);
+  }
+
+  // Create a new account with this threshold key
+  const accountCreateTransaction = await new AccountCreateTransaction()
+    .setKey(thresholdKey)
+    .setInitialBalance(0)
+    .execute(client);
+
+  console.log(await accountCreateTransaction.getReceipt(client)._accountId);
```

Comment:
Should  use `accountCreateTransaction.getReceipt(client).getAccountId()` since `TransactionReceipt._accountId` is meant to be private and for internal use only.

Reply:
Hey @danielakhterov no reason for the client config being on two lines, I was just trying to match [this example](https://github.com/hashgraph/hedera-sdk-js/blob/c7d5a38a2ebc8a66e84d248a4baae5fa0d42e70b/examples/get-account-balance.js#L11) which was added by someone on your team. Do you want me to update? I don't mind 😄 

c4f66eabad1901ba5df52604b5be360a69608e4a should resolve the `.getAccountId()` issue. Thanks for the QA!